FlyingSocks/Tests/SocketAddressTests.swift

```diff
@@ -180,4 +180,121 @@ final class SocketAddressTests: XCTestCase {
             XCTAssertEqual($0, .unsupportedAddress)
         }
     }
+
+    func testSockaddrInToStorageConversion() throws {
```

Comment:
Do you think the existing tests within SocketAddressTests already cover these same things?
- `testAddress_DecodesIP4()`
- `testINET_ThrowsInvalidAddress_WhenFamilyIncorrect()`
- `testAddress_DecodesIP6()`
- `testINET6_ThrowsInvalidAddress_WhenFamilyIncorrect()`
- `testUnix_IsCorrectlyDecodedFromStorage()`
- `testUnix_ThrowsInvalidAddress_WhenFamilyIncorrect()`

The existing `makeStorage()` is tested within the "decode" tests are more of an encode/decode, they start with an address, call `makeStorage()` then convert back to the original address to assert.
-

Reply:
Since they didn't run red, they should definitely be modified, if you think I'd better merge the tests. Since they run quite quickly, I thought it was no problem to validate the storage specific ones separately.

Tastes and slaps, I don't like overlapping test cases, however as our example shows, they can mask underlying-unit errors.

Reply:
Also, if we go along with the current solution, I would like to note that I have worded the tests differently, if you like, I would be happy to reword them to the existing style (with underscores).